### DISCUSSION TITLE
BIRT 4.8.0 source code

### DISCUSSION THREAD

**Discussion by syc795** [eclipse-birt/birt#715] [Q&A] [answered]
I see the version tag till BIRT_4_7_0_Release201706222054 . I want 4.8.0 source code, so where I can get it, any specific date? branch?

**Comment by wimjongman** [comment, marked as answer]
Thanks for noticing. I created a new tag:

https://github.com/eclipse/birt/releases/tag/BIRT_4_8_0_Release_201806261756
